core/src/main/kotlin/fr/sncf/osrd/api/stdcm/STDCMEndpoint.kt

```diff
@@ -136,8 +137,8 @@ fun makeTrainSchedule(
     trainStops: List<TrainStop>
 ): StandaloneTrainSchedule {
     val mutTrainStops = ArrayList(trainStops)
-    // Force the train to end its path at speed=0 and on closed signal
-    mutTrainStops.add(TrainStop(endPos, 0.1, true))
+    // Force the train to end its path at speed=0 and on closed signal with short slip distance
+    mutTrainStops.add(TrainStop(endPos, 0.1, SHORT_SLIP_STOP))
```

Comment:
Does it make sense as "default value" for a generic end-of-path stop? `STOP` would feel more standard

Reply:
As discussed, this was refined to use the most restrictive, and this will be documented after this PR: https://github.com/OpenRailAssociation/osrd-website/pull/240/files